com.tobagin.whakarere.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.tobagin.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+#rename-desktop-file: whakarere.desktop
+#rename-icon: whakarere.png
+
+finish-args:
+  - --share=ipc     # D-Bus
+  - --share=network  # If your script needs network
+  - --socket=x11    # If your script has a GUI
+  - --socket=wayland # If your script has a GUI
+  - --socket=pulseaudio # If your script needs sound
+  - --socket=session-bus # If your script needs D-Bus
+  - --device=dri     # If your script uses 3D acceleration from the GPU
+  #- --filesystem=host/usr/share/fonts:ro
+
+modules:
+  - name: Whakarere
+    buildsystem: simple
+    build-commands:
+      - install -D whakarere.py /app/bin/whakarere
+      - install -Dm644 whakarere.png /app/share/icons/hicolor/128x128/apps/com.tobagin.whakarere.png
```

Comment:
this is not the size of the icon.

Reply:
I've changed to reflect the correct size, its 512x512